kubeconfig_uat_zwm5s_60d5f81ac6c5fc15c236362b677b1a4c18f9b06e.yaml

```diff
@@ -0,0 +1,18 @@
+    client-key-data: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb2dJQkFBS0NBUUVBcG5DeVNSOWRqNVF4UHVGVXpMYVJZeGFHeHFjK3FGcE9UWW8xM1VBZFJqME5qNE9zClhFbjdSUjhod0NXYkZkdDJLRHVOQXZ2eXB1VmVkOXR0TnI2bVQvY3phckREUXhFUmlpdmhsRTU2ajJSQmRGVGwKNnJibDBqeFgwVE90QWNSbklYRVRzSzYrUy9sQlR6N0JjMm5Zc1BoeERpZTB5ZkZIQ3I5VE5FaXc1ejc4VlhNSgpuWVFiV1ZOckxwajZVb3lGRXlXZGQ3QS95U2lFb1EwMWNVT0tDMUlQNzd4c1hqbE51emhYc094OXZiWU1tWUY3CkRnTDBZVTBsWlpmdExjakplaVpGWHZqaVdGUDdicjVHZXBRb2YvS0tab1Y4UzR0THpidEtWNFc2SnMvRTI1cUIKSHhCaVZqUi9weDZ4b29senFoZDRhamNKdUJLR3cxMzJubmduWXdJREFRQUJBb0lCQUYwbGN0RnphVVFDTm5FdgpYK3JWUFpaTHRBK1c1MzBHVlE0V0RFRVBNWkx1ZGhsY0ltamZ5bXlMbUxMVkdPN3BqRDNGM2d2T2ZxVWlkblR6CmF6S1pEYkJ3OHVpQmxqV0JadVNCM0ZQSWU5U2wwZS9OMTRJODYyOTE3OXVYWlBvNU5WSkUyelVsQmtDMGRQVnAKZEpBbnEycVhENFpSZUY4UDFhZFJIYnFEOStUaHRvZElGbkYzcTBOajZTeW9xRmVxWjBhSWRqWDRMMzRwTEdkcgovOUpjdGZrSU9BQ241NTc3YWJHVGtycU42cVovK09xV2hrTTQ2d2hVNFR6SGlZdnBialRWamZJelNIci9NZFc5CjlMRWdPdmtNZWxYaVNRbU1HTzhFWHFHcWppRDdKZmVnS1lFcXUxcEJIUXhXbitNQXM0eVY0b3ZDU3RvcEVqK0QKRHBlSHJjRUNnWUVBM0JmSUZ5R3FDdTJuYjdpN0VmTHplMHpJZVltMmNxaTZvMitreTdYNktodk1VZUZiRDVQRgpUUDhDNzZ3WHQ2R1BOQWNYTlN3dEp3aHBPdC9jV3BDVFFCVlVjREdXRHZiTEFvZHN1cHI1eUNLcHFVVHhLeklRCjVtSkhEUlQ4K3VlL1VMYjFNTFVZV2R2d252QVlDQXRSQ2V6ZURZdGZPYkZqclgxYVlEV2ovV0VDZ1lFQXdaZ1oKOU9pdVRXYzBGNmV3cjJ3dFlhRjB5alVqdTZXTzZVNXc5R1FLcjdYTU43eWVkazlaYVhKdUFzTGpVMDg0RGk0bApnR1ZpRnFCL1NKc3U4TDNCbnhsc3l6OGw3Y0dDb1BjQ3ZhYzJucVRHc2c1NG53MDZQM3ZEaS94Q2ZVTVc5MXZqCjc3b01ERFpBRHNXV21YRWlXbFVtMUVhVUpVbnZOUU00d0prOU4wTUNnWUJvZE9FQ0dYZ2F0N0EvMWV6ekY5dDMKS1huOE5OL2lpeklKWDNOWE80Yi9kVElEcW5FTmM1U3dkbGR1YUw3WTZGU08zTTZtN2RDQU5mMjdjNE9oZC94cQo3YlNsaXVNdE52UWl2eU5iYlRJZlNsUTdnZnowQTA4Q0NLS3dFc1grVHY3OXo5ZU94bVJJUVpweDFJR1RUOHVtCmFrVDVLS3lKTlNlWnIyUzFSTUFRQVFLQmdBdVVYWGtZc2RSUFovNUNPWDlFcVRJOXhnNlNQeURxTmpJZVR3TlAKT0pScCsxMGFHeUJZZnJDZjk1U1NzNlk0QnRKU1VFdnR4SUExK1djT1lUQmxLVCtxOWFieWtBSTRvT2VmM3JvWgpWbHNHNEJ1bVlmQ2hpZXBpTE1BaXhwZFhlL2d2dHVLUDdraU10aWdzQ1dZRUxsblkzZ20wYndnTDlCTG5ybTFXCnZTS2hBb0dBWHZTRjB3cjFrV2VXRHdhcWZ4T2FyVGp2TUgyNEh4d2gzQUk0N3l0ZDhtV0V3Q3JXTWRSNnBuMVgKVkRjTHRxRXNJclFiL2RZVW9ITlVlRlowc2dLQjM0L3lXdlJXZTg3eGxZU0FxRk5UcU9RM1dnemNMYkozRjdpMgpIbU5hZUF4TU5aVUlRNWJubDdhSDJyVG90bFVGaXpPNHJ6YXVsdWJ1UEx0b3FHalBkSDQ9Ci0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0t
```

Comment:
Why do we commit private keys here?

Reply:
Accidentally. As both the key-pair and the clusters are short-lived, no danger.